### ISSUE TITLE
Login button broken when using an ad-blocker

### ISSUE THREAD

**Issue by marco-c** [mozilla/distribution-viewer#172]
I mentioned this to Rob, who isn't able to reproduce.

Perhaps the issue is due to one of the filter lists I'm using:
AdBlock Warning Removal List
EasyPrivacy
AakList (Anti-Adblock Killer)
Fanboy's Annoyance List
EasyList Italy + EasyList
facebook annoyances blocker

**Comment by openjck** [comment]
Do you also have Disconnect, Privacy Badger, or other privacy add-ons installed by any chance?

**Comment by openjck** [comment]
I think disabling third-party cookies or using tracking protection in Firefox also causes issues.

**Comment by marco-c** [comment]
No, only AdBlock Plus. If I disable it, I see the button. If I enable it, I don't see the button.

I do have tracking protection enabled, but the button works with it if I disable AdBlock Plus.
